examples/run_CPMG.py

```diff
@@ -1,125 +1,126 @@
-"""
-CPMG
-====
-
-This script will perform a standard CPMG experiment. 
-In order to form a symmetric echo, a padding time is added before 
-and after your tau through a series of delays. 
-"""
```

Comment:
why remove this?  I think that you need to generally read over the changes that you are proposing here.